lib/cli/deploy.rs

```diff
@@ -230,7 +241,12 @@ pub fn make_transfer(
         payment_params,
         true,
     )?;
-    crate::output_deploy(output, &deploy).map_err(CliError::from)
+    #[cfg(feature = "std-fs-io")]
+    {
+        let output = parse::output_kind(maybe_output_path, force);
+        let _ = crate::output_deploy(output, &deploy).map_err(CliError::from);
```

Comment:
```suggestion
        crate::output_deploy(output, &deploy).map_err(CliError::from)?;
```

Reply:
Thanks should be fixed in

https://github.com/casper-ecosystem/casper-client-rs/commit/b4a8f436f6d31c6cc38a1ad93843d9523279dd2c